### ISSUE TITLE
Social icons are broken in Safari

### ISSUE THREAD

**Issue by vikmarino** [DigitalMindCH/twister-jekyll-theme#2]
This is what I see when open [your site](http://digitalmind.ch/themes/twister-jekyll-theme/demo/) in Safari **8.0.7 (10600.7.12)**
![screen shot 2015-09-03 at 0 18 05](https://cloud.githubusercontent.com/assets/7696194/9646171/60ecac42-51d1-11e5-991d-7e7341b396fe.png)

**Comment by vikmarino** [comment]
And by the way. How can we add github icon for instance? I suggest to add this section to **Docs**

**Comment by halfapx** [comment]
Hmm.. everything looks fine on our side, if we open the site in Safari... Does it still look like this?
The Icons are added using [Font Awesome](http://fontawesome.io/). Add an `i`-tag with the right class and then just wrap it in a link.

**Comment by vikmarino** [comment]
Yes, I have just checked it again and it still shows only two icons. The links are there though, but their icons aren't visible.

**Comment by halfapx** [comment]
Okay, very very mystical. I will ask some more Mac-Users if any of them have the same issue and try to check it on their computers.
